build/yamls/base/conf/antrea-agent.conf

```diff
@@ -31,8 +31,8 @@ featureGates:
 # Enable controlling SNAT IPs of Pod egress traffic.
 #  Egress: false
 
-# Enable flexible IPAM mode for Antrea. This mode allows to assign IP Ranges to Namespaces,
-# Deployments and StatefulSets via IP Pool annotation.
+# Enable AntreaIPAM. AntreaIPAM is required by the bridging mode and can assign IPs to Pods in
+# bridging mode.
```

Comment:
Shouldn't we still mention what is purpose for this feature (to allocate IPs via pools?)

Reply:
Sure. Let me add that.